spring-cloud-contract-verifier/src/main/java/org/springframework/cloud/contract/verifier/messaging/amqp/SpringAmqpStubMessages.java

```diff
@@ -84,10 +86,14 @@ public SpringAmqpStubMessages(RabbitTemplate rabbitTemplate, MessageListenerAdap
 
 	@Override
 	public void send(Message message, String destination) {
-		if (this.messageListenerAdapter == null) {
-			throw new IllegalStateException("no MessageListenerAdapter wired - cannot send message");
+		List<SimpleMessageListenerContainer> listenerContainers = this.messageListenerAccessor.getListenerContainersForDestination(destination);
+		if (listenerContainers.isEmpty()) {
```

Comment:
I agree - the other integrations seem to throw an exception in such a case - I will change this one.